index.bs

```diff
@@ -595,6 +595,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
         to |authenticator|'s attachment modality, [=iteration/continue=].
     1. If {{AuthenticatorSelectionCriteria/requireResidentKey}} is set to |true| and the |authenticator| 
         is not capable of storing a [=Client-Side-Resident Credential Private Key=], [=iteration/continue=].
+    1. If {{AuthenticatorSelectionCriteria/requireUserVerification}} is set to |true| and the
```

Comment:
This needs to also get passed on to the authenticator, which needs to be specified to use it to ensure that future `get()` calls are authorized by the same user as authorized the `create()` call.